extensions/wikia/AdEngine/js/video/player/ui/panel.js

```diff
@@ -0,0 +1,46 @@
+/*global define, setTimeout*/
+define('ext.wikia.adEngine.video.player.ui.panel', [
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (doc, log, win) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "win". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

[project-id]: wikia:app